index.d.ts

```diff
@@ -98,6 +98,26 @@ declare module "webhook-discord" {
      */
     public setTime(timestamp?: number): MessageBuilder;
 
+    /**
+     * Set the title for the discord hook
+     * @param title the webhooks´ title
+     */
```

Comment:
This should use the documentation from the JavaScript file.
```suggestion
    /**
     * Set the title of the embed
     *
     * @param {string} title The title to use
     */
```